Maple2.File.Parser/Xml/AI/NpcAi.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using System.Xml.Serialization;
+
+namespace Maple2.File.Parser.Xml.AI;
+
+[XmlRoot("npcAi")]
```

Comment:
We generally include a comment here that quantifies the files that this schema applies to. In this case maybe something like:

```cs
// ./data/xml/ai/%s.xml
```
or
```cs
// ./data/xml/ai/**/%s.xml
```